### PR TITLE
LPD-23945, LRCI-4187 | Add playwright tests to release tester

### PR DESCRIPTION
**Pull request by john-co**
**JIRA Ticket**
https://liferay.atlassian.net/browse/LPD-23945
https://liferay.atlassian.net/browse/LRCI-4187

### CODE REVIEW

**Comment by john-co** [comment]
cc/ @brittneyq @kenjiheigel we plan to add all playwright tests to release tester. Do we need to wait for any other ci infra changes to merge first? ie: https://liferay.atlassian.net/browse/LRCI-4187

**Comment by john-co** [comment]
I tried implementing the `bundle` batch ant script. Let me know how it looks and if there's a good way I can test the script somehow.

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:playwright

File: build-test-batch.xml [hunk 1]
```diff
@@ -781,6 +781,180 @@ mariadb.executable=${mariadb.executable}]]></echo>
...
+
+							<property name="artifact.version" value="1.0.0-TEST_ARTIFACT" />
+
+							<copy-artifact-to-local-cache artifact.file="${test.app.server.lib.portal.dir}/portal-impl.jar" artifact.name="com.liferay.portal.impl" artifact.version="${artifact.version}" />
+							<copy-artifact-to-local-cache artifact.file="${test.app.server.lib.portal.dir}/portal-kernel.jar" artifact.name="com.liferay.portal.kernel" artifact.version="${artifact.version}" />
+							<copy-artifact-to-local-cache artifact.file="${project.dir}/portal-test/portal-test.jar" artifact.name="com.liferay.portal.test" artifact.version="${artifact.version}" />
+							<copy-artifact-to-local-cache artifact.file="${test.app.server.lib.portal.dir}/util-bridges.jar" artifact.name="com.liferay.util.bridges" artifact.version="${artifact.version}" />
+							<copy-artifact-to-local-cache artifact.file="${test.app.server.lib.portal.dir}/util-java.jar" artifact.name="com.liferay.util.java" artifact.version="${artifact.version}" />
+							<copy-artifact-to-local-cache artifact.file="${test.app.server.lib.portal.dir}/util-taglib.jar" artifact.name="com.liferay.util.taglib" artifact.version="${artifact.version}" />
```
Comment (john-co) [inline build-test-batch.xml:947]: I didn't get a chance to study what these lines do, so I'm not sure how necessary these are for playwright tests.

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:playwright

**Comment by john-co** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
@john-co, if you would like to test this on release tester it would be best use a release branch like the one here: https://test-1-1.liferay.com/job/test-portal-release/4393/parameters/ 

1. Fetch that tag https://github.com/brianchandotcom/liferay-portal-ee/tree/7.4.13-u118 locally into a new branch
2. cherry pick your changes onto that new branch
3. run release tester with the same bundle parameters here: https://test-1-1.liferay.com/job/test-portal-release/4393/parameters/, but change the parameters TEST_PORTAL_RELEASE_GIT_ID to your new branch's latest git ID after the cherry-pick, TEST_PORTAL_USER_BRANCH_NAME to your new branch name, TEST_PORTAL_USER_NAME as your github username, and CI_TEST_SUITE to your test suite (I suggest maybe using a different portal-release test suite incase or others may be confused why a full portal-release suite was run... perhaps making a new test suite like portal-release-playwright and adding that to test.properties and ci.properties)

**Comment by john-co** [comment]
Thanks @brittneyq !

**Pull request closed without merging**